### PR TITLE
Use Connection() from util package

### PR DESCRIPTION
**Pull request by jsafrane**
The result should be the same as in https://github.com/kubernetes-csi/external-attacher/pull/123

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -185,55 +184,18 @@ func logGRPC(ctx context.Context, method string, req, reply interface{}, cc *grp
...
-	}
-	name := rsp.GetName()
-	if name == "" {
-		return "", fmt.Errorf("name is empty")
-	}
-	return name, nil
+	return connection.GetDriverName(ctx, conn)
 }
 
 func getDriverCapabilities(conn *grpc.ClientConn, timeout time.Duration) (sets.Int, error) {
```
Comment (msau42) [inline pkg/controller/controller.go]: and update getcapabilities?

Comment (jsafrane) [reply pkg/controller/controller.go]: added

**Comment by msau42** [comment]
cc @lpabon

File: pkg/controller/controller.go [hunk 2]
```diff
@@ -248,30 +210,16 @@ func getDriverCapabilities(conn *grpc.ClientConn, timeout time.Duration) (sets.I
...
-		if cap == nil {
-			continue
-		}
-		service := cap.GetService()
-		if service == nil {
-			continue
-		}
-		switch service.GetType() {
+	for cap := range pluginCaps {
+		switch cap {
```
Comment (msau42) [inline pkg/controller/controller.go]: Is mixing plugin and controller capabilities into the same set correct?  Could you have conflicts in values?

Comment (jsafrane) [reply pkg/controller/controller.go]: `PluginCapability_*` and `ControllerCapability_*` are part of the same enum defined in the provisioner, they don't come from CSI:

https://github.com/kubernetes-csi/external-provisioner/blob/e63995ffb48736dc6275b5e8702e2dc8868c57ef/pkg/controller/controller.go#L160-L165

Comment (msau42) [reply pkg/controller/controller.go]: Thanks for pointing that out. Is there a reason why it needs to have one more level of translation from CSI capabilities?

Comment (jsafrane) [reply pkg/controller/controller.go]: Can we sort it out in a separate PR? I'd like to cache the capabilities so provisioner does not load them from the driver every time it needs them. Refactoring of the enum would be better there.

Comment (msau42) [reply pkg/controller/controller.go]: Sure do you want to use csi-lib-utils 0.4.0-rc1 here?

Comment (jsafrane) [reply pkg/controller/controller.go]: Updated to 0.4.0-rc1

**Pull request merged**